x-pack/plugins/viz_editor/pie_chart_plugin/pie_chart_plugin.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { EuiSuperSelect } from '@elastic/eui';
+import clone from 'lodash-es/clone';
```

Comment:
I think you can just `import { clone, setWith } from 'lodash-es/setWith'`, and it should still tree-shake fine: https://medium.com/@martin_hotell/tree-shake-lodash-with-webpack-jest-and-typescript-2734fa13b5cd